syntax = "proto2";
package mmo.proto;

message VendorItemEntry {
	required uint32 item = 1;
	optional uint32 maxcount = 2;
	optional uint64 interval = 3;
	optional uint32 extendedcost = 4;
	optional bool isactive = 5 [default = true];
}

message VendorEntry {
	required uint32 id = 1;
	required string name = 2;
	repeated VendorItemEntry items = 3;
}

message Vendors {
	repeated VendorEntry entry = 1;
}
